servers/rendering/rasterizer_rd/rasterizer_scene_high_end_rd.cpp

```diff
@@ -2700,6 +2854,14 @@ RasterizerSceneHighEndRD::RasterizerSceneHighEndRD(RasterizerStorageRD *p_storag
 			defines += "\n#define MAX_GI_PROBES " + itos(scene_state.max_gi_probes) + "\n";
 		}
 
+		{ //decals
+			scene_state.max_decals = MIN(1024 * 1024, uniform_max_size) / sizeof(DecalData); //1mb of decals
```

Comment:
Should this be a project setting?

Reply:
well, thats a huge number, At some point when we add the tweaks category of the project settings as we discussed we could expose this, otherwise its probably not worth it for now